indexer/services/comlink/__tests__/controllers/api/v4/compliance-v2-controller.test.ts

```diff
@@ -231,7 +236,12 @@ describe('ComplianceV2Controller', () => {
   });
 
   describe('POST /geoblock', () => {
```

Comment:
nit: Add test case of unexpected `ONBOARD` action, leading to error-log + no-op?

Reply:
done